config/rbac/service_account.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: controller-manager
```

Comment:
Although the name here is `controller-manager`, when created I see a SA with the name `etcd-druid-controller-manager`
Is this expected?

Reply:
This is expected as all resources will be prefixed with `etcd-druid-` as defined in the `config/default` kustomization. In general it is a good practise to use generic names such as `controller-manager` in the sub-kustomizations and prefix is later. E.g. a user might have his own default and overwrite the `namePrefix: abcd-` and hence will end up with a `abcd-controller-manager SA.

Reply:
Ah, that clarifies things
Thank you